ksqldb-rest-app/src/main/java/io/confluent/ksql/rest/server/KsqlRestApplication.java

```diff
@@ -1056,6 +1061,72 @@ private void registerCommandTopic() {
 
     final String commandTopic = commandStore.getCommandTopicName();
 
+    // migration code
+    if (checkMigrationConditions(commandTopic)) {
+      log.warn("Migrating command topic from the service context Kafka to the command "
+          + "producer/consumer Kafka.");
+      KsqlInternalTopicUtils.ensureTopic(
+          commandTopic,
+          ksqlConfigNoPort,
+          internalTopicClient
+      );
+
+      try {
+        CommandTopicMigrationUtil.commandTopicMigration(commandTopic, restConfig, ksqlConfigNoPort);
+      } catch (final Exception e) {
+        throw new KsqlException("Error migrating command topic", e);
```

Comment:
why do we rewrap here?

Reply:
It's not really necessary, I'll get rid of the try catch